### PR TITLE
Video listing

### PR DESCRIPTION
**Pull request by tanishk15sharma**
**VIDEOS LISTING PAGE**
_Style:_
- Sidebar 
- Navigation 
- Videos-wrapper 
- Video card

_Functionility:_
- Storing data in backend
- Fetching data from backend using `axios`
- Rendering data in UI
- Link to player

### CODE REVIEW

File: src/components/sidebar/Sidebar.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.sidebar {
+  background-color: var(--background-secondary-color);
+  position: fixed;
+  height: 100%;
+  top: 0;
+  display: flex;
+  flex-direction: column;
+  width: 80px;
+  text-align: center;
+  /* display: none; */
```
Comment (prathmeshjagtap) [inline src/components/sidebar/Sidebar.css:10]: you can remove commented code

File: src/components/videos-listing/VideosListing.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.relative {
+  position: relative;
+}
+
+.hero-div {
+  background: var(--background-primary-color);
+  /* width: 100%; */
```
Comment (prathmeshjagtap) [inline src/components/videos-listing/VideosListing.css:7]: remove commented  code if you are not going to use it in future

**Review by prathmeshjagtap — APPROVED** [review]
code is Well structured and clean, just remove the commented code if not required

File: src/components/sidebar/Sidebar.css (hunk 1 above)
Comment (ArjunGTX) [reply src/components/sidebar/Sidebar.css:10]: +1

File: src/components/videos-listing/SuggestionChips.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
+.chips-div {
+  padding: 0.8rem 2rem;
+  background: var(--background-secondary-color);
```
Comment (ArjunGTX) [inline src/components/videos-listing/SuggestionChips.css:3]: Isn't this variable name too long? can't it be `bg-secondary-color` or just `bg-secondary`

File: src/components/videos-listing/VideosListing.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+}
+.left {
+  width: 95%;
+  margin-left: auto;
+  /* margin-right: 0; */
```
Comment (ArjunGTX) [inline src/components/videos-listing/VideosListing.css:12]: Remove commented code

File: src/index.css [hunk 5]
```diff
@@ -11,13 +11,15 @@
   list-style: none;
 }
 :root {
-  --primary-color: #fff;
+  --primary-color: #f5f5f4;
+  --secondary-text: #aaaaaa;
 
   /* --secondary-color: #3e4152; */
```
Comment (ArjunGTX) [inline src/index.css:17]: Same as above

**Review by ArjunGTX — COMMENTED** [review]
Looks good in Overall. Suggested some minor changes.

File: src/components/sidebar/Sidebar.css (hunk 1 above)
Comment (tanishk15sharma) [reply src/components/sidebar/Sidebar.css:10]: Yes I will remove the comments soon .

File: src/components/videos-listing/SuggestionChips.css (hunk 3 above)
Comment (tanishk15sharma) [reply src/components/videos-listing/SuggestionChips.css:3]: you are right , I will fix this

File: src/index.css (hunk 5 above)
Comment (tanishk15sharma) [reply src/index.css:17]: yes I will remove all the comments .

**Pull request merged**